proposals/lip-0068.md

```diff
@@ -182,7 +182,7 @@ This LIP results in a hard fork as nodes following the proposed protocol will re
 
 ## Reference Implementation
 
-TBD
+[Update lisk-chain to support the on-chain architecture #5609](https://github.com/LiskHQ/lisk-sdk/issues/5609)
```

Comment:
I think the Transaction class should be linked from here, i.e. https://github.com/LiskHQ/lisk-sdk/blob/development/elements/lisk-chain/src/transaction.ts. I don't see how the PR you linked is connected to the updated trs schema.